docs/architecture/adr/0021-console-logging.md

```diff
@@ -0,0 +1,114 @@
+---
+adr: "0021"
+status: In progress
+date: 2023-07-13
+tags: [server]
+---
+
+# 0021 - Logging with the Console
+
+<AdrTable frontMatter={frontMatter}></AdrTable>
+
+## Context and Problem Statement
+
+As the server platform has matured so have the various logging extensions to support additional use
+cases and customer requests. [Serilog][serilog] is in place via shared logic for all services,
+initiated at startup, and over time additional its "sinks" with their needed configuration have been
+added as dependencies leaving this core logging layer with a growing list of conditions on how and
+when to use certain types of structured logging. Maintenance needs have grown to keep sink
+dependencies up to date and more are desired to be added. Some of the presently-available sinks have
+very little use and / or better alternatives now exist.
+
+## Considered Options
+
+:::note
+
+Bitwarden uses [Datadog][dd] as its monitoring tool and desires to increase its usage by engineers
+across the board to improve what we deliver.
+
+:::
+
+- **Maintain current logging options** - Support what is available today for logging methods and
+  expect those running the platform to configure what they need outside of it for log collection.
+- **Extend the plaform to specifically support Datadog** - A Serilog sink [exists][ddsink] and the
+  platform can send logs directly to Datadog.
+- **Consolidate logging providers** - Announce deprecation and migration plans for sinks not aligned
+  with core needs and center on the console for logs.
```

Comment:
```suggestion
- **Consolidate logging providers** - Announce deprecation and migration plans for sinks not aligned
  with core needs and center on the stdout for logs.
```

Reply:
I might even got a step further and call this option "Adopt cloud native logging practices - Deprecate existing sinks and align on stdout & file based logging."